controllers/install/lookout_controller_e2e_test.go

```diff
@@ -0,0 +1,55 @@
+package install
+
+import (
+	"io"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/armadaproject/armada-operator/controllers/utils"
+)
+
+var lookoutYaml = `apiVersion: install.armadaproject.io/v1alpha1
+kind: Executor
+metadata:
+  labels:
+    app.kubernetes.io/name: lookout
+    app.kubernetes.io/instance: bincoulars-sample
+    app.kubernetes.io/part-of: armada-operator
+    app.kubernetes.io/created-by: armada-operator
+  name: lookout-e2e
+spec:
+  image:
+    repository: test-lookout
+    tag: latest
+  applicationConfig:
+    server: example.com:443
+    forceNoTls: true
+    toleratedTaints:
+      - key: armada.io/batch
+        operator: in
+`
+
+var _ = Describe("Armada Operator", func() {
+	When("User applies Binoculars YAML using kubectl", func() {
```

Comment:
Change the names of the tests.  You use binculars and executor rather than lookout.